lib/iris/cube.py

```diff
@@ -716,9 +714,8 @@ def __init__(self, data, standard_name=None, long_name=None,
         if isinstance(data, six.string_types):
             raise TypeError('Invalid data type: {!r}.'.format(data))
 
-        self._dask_array = None
-        self._numpy_array = None
-        self.data = data
+        # Initialise the cube data manager.
+        self._data_manager = DataManager(data, realised_dtype=dtype)
```

Comment:
What if the `data` attribute is lazy? Does `realised_dtype` then imply the target dtype upon realisation?